bundle/edu.gemini.wdba.xmlrpc.server/src/test/java/edu/gemini/wdba/tcc/Flamingos2SupportTest.java

```diff
@@ -71,14 +71,14 @@ public void testWavelength() throws Exception {
         // Use the disperser wavelength in spectroscopy mode with no filter
         flam2.setFilter(Filter.OPEN);
 
-        Pair[] dtA = new Pair[] {
-            new Pair<Disperser, String>(Disperser.R1200JH, "1.39"),
-            new Pair<Disperser, String>(Disperser.R1200HK, "1.871"),
-            new Pair<Disperser, String>(Disperser.R3000,   "1.65"),
+        final Pair[] dtA = new Pair[] {
```

Comment:
Very annoying. Even doing an `Arrays.asList(new Pair<...>(...), ...)` fails with "generic array creation" because this method relies on `Collections.toArray` to work, and cannot thus work with arrays of generics. The only solution, I think, would be to specifically create an `ArrayList<Pair<...>>` and then add the objects one at a time, which I think we'll not worry about for now.